app/views/pages/home.html.erb

```diff
@@ -6,6 +6,5 @@
   </div>
   <%= link_to "I want to find a location", places_search_path, class: "btn-home" %>
   <%= link_to "I want to start a lineup", places_map_path, class: "btn-home" %>
-<!--   <div class="footer"></div>
- --></div>
+  <%= render 'shared/footer' %>
```

Comment:
please add the footer to the application.html.erb instead so it will be in all pages

Reply:
Right! ;) it is done!